langchain/embeddings/openai.py

```diff
@@ -338,8 +338,8 @@ def _get_len_safe_embeddings(
                 disallowed_special=self.disallowed_special,
             )
             for j in range(0, len(token), self.embedding_ctx_length):
-                tokens += [token[j : j + self.embedding_ctx_length]]
-                indices += [i]
+                tokens.append(token[j : j + self.embedding_ctx_length])
```

Comment:
should be extend, no? same in L422

Reply:
The original code creates a 1-element list and then adds all the elements (one) of this list to tokens. So instead I just appended that one element. I don't know if there is a bug in the original code and it should add every byte of the encoded token to the tokens array instead, but that would seem odd to me?

Reply:
oh you're right, misread! thought was adding ` += token[j : j + self.embedding_ctx_length]` directly